Extensions/TweenBehavior/JsExtension.js

```diff
@@ -148,6 +146,54 @@ module.exports = {
       .addIncludeFile('Extensions/TweenBehavior/tweentools.js')
       .setFunctionName('gdjs.evtTools.tween.tweenCamera');
 
+    extension
+      .addAction(
+        'TweenCameraZoom',
+        _('Tween the camera zoom'),
+        _('Tweens the camera zoom from the current one to a new one.'),
+        _(
+          'Tween the zoom of camera on layer _PARAM3_ to _PARAM2_ for _PARAM4_ms with easing _PARAM5_ as _PARAM1_'
+        ),
+        _('Scene Tweens'),
+        'JsPlatform/Extensions/tween_behavior24.png',
+        'JsPlatform/Extensions/tween_behavior32.png'
+      )
+      .addCodeOnlyParameter('currentScene', '')
+      .addParameter('string', 'Tween identifier', '', false)
+      .addParameter('expression', 'Target zoom', '', false)
+      .addParameter('layer', 'Layer', '', true)
+      .addParameter('expression', 'Duration', '', false)
```

Comment:
I've just seen we're missing translation markers everywhere 😱

(Tween identifier...)